src/main/java/seedu/duke/Ui.java

```diff
@@ -0,0 +1,204 @@
+package seedu.duke;
+
+import seedu.exceptions.InvalidTaskNameException;
+import seedu.task.Deadline;
+import seedu.task.Event;
+import seedu.task.TaskList;
+import seedu.task.Todo;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ *  This UI class contains methods that print out chatbot
+ *  messages in the console to the user.
+ */
+public class Ui extends TaskList {
+    private final String line = "_____________________________________________________";
```

Comment:
According to the coding style for this module, 'line' is a constant and hence should be named LINE instead!